app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.scss

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.tabs {
+  display: flex;
+  max-width: 800px;
+
+  @media (max-width: $SCREEN_MD_MAX) {
+    max-width: 570px;
+  }
+}
+
+.side-block {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 120px;
+  height: 58px;
+  background-color: $COLOR--black-1;
+  border-bottom: 4px solid $COLOR--gray-47;
+  border-radius: 4px;
+  cursor: pointer;
+
+  @media (max-width: $SCREEN_MD_MAX) {
+    width: 80px;
+  }
+
+  &-title {
+    padding-bottom: 3px;
+    color: $COLOR--white-two;
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 16px;
+    letter-spacing: 0.4px;
+  }
+
+  &-text {
+    color: $COLOR--dirty-gray;
+    font-size: 7px;
+    font-weight: 700;
+    line-height: 10px;
+    letter-spacing: 0.4px;
+    text-transform: uppercase;
+  }
+
+  &-active {
+    position: relative;
+    color: $COLOR--topaz;
+    background-color: $COLOR--topaz-disabled;
```

Comment:
Please use corresponding color for background from markup
`background: #00505D;`
Also, property `color` doesn't apply for active item.